sunpy/coordinates/sun.py

```diff
@@ -73,6 +74,38 @@ def sky_position(t='now', equinox_of_date=True):
     return ra, dec
 
 
+# Time in Julian Days of the start of CR1
+_FIRST_CROT_JD = 2398167.4
+# Length of a carrington rotation in days
+_CARRINGTON_ROTATION_PERIOD = 27.2753
+
+
+def carrington_rotation_time(crot):
+    """
+    Return the time of a given Carrington rotation.
+
+    The round-trip from this method to `carrington_rotation_number` has
+    absolute errors of < 0.11 seconds.
+
+    Parameters
+    ----------
+    crot : float
+        Carrington rotation number. Can be a fractional cycle number.
+    """
+    estimate = (_CARRINGTON_ROTATION_PERIOD * (crot - 1)) + _FIRST_CROT_JD
+
+    # The above estimate is wrong (see comments below in carrington_rotation_number),
+    # so put the estimate into carrington_rotaiton_number to get the fraction correction needed
+    def refine(estimate):
+        crot_estimate = carrington_rotation_number(t=Time(estimate, format='jd'))
```

Comment:
What does `scale='tt'` do?

Reply:
When precision is important, you be careful with the time scale, whether it's Terrestrial Time (TT), Coordinated Universal Time (UTC), or something else.  TT (or the nearly equivalent Barycentric Dynamical Time (TDB)) is generally a more common scale than UTC for astronomical purposes, particularly over long time scales, because it's not tied to the changing rotation of the Earth.

It's for the same reason that you should always cast a `Time` object to an explicit scale before calling `.jd` (e.g., `.tt.jd` or `.utc.jd`).

Reply:
https://en.wikipedia.org/wiki/Time_standard